corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -4,8 +4,9 @@ if (typeof cloudCare === 'undefined') {
 
 cloudCare.EMPTY = '---';
 
-var _caseListLoadError = function (response) {
+var _caseListLoadError = function (model, response) {
```

Comment:
yea, backbone usually passes model as first arg: http://backbonejs.org/#Model-fetch. 
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/lib/backbone.js#L1226. also tested this via a breakpoint